2015-11-20 23:12:01,479 - [INFO] - from play in pool-6-thread-3 
Listening for HTTP on /0:0:0:0:0:0:0:0:9000

2015-11-20 23:12:27,997 - [INFO] - from play in play-internal-execution-context-1 
datasource [jdbc:h2:file:data/db;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-11-20 23:12:28,251 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:h2:file:data/db

2015-11-20 23:12:29,313 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2015-11-20 23:12:29,613 - [INFO] - from application in play-akka.actor.default-dispatcher-7 
http://localhost:9034/climateService/getAllClimateServices/json

2015-11-20 23:16:32,932 - [INFO] - from play in play-internal-execution-context-2 
datasource [jdbc:h2:file:data/db;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-11-20 23:16:32,938 - [INFO] - from play in play-internal-execution-context-2 
database [default] connected at jdbc:h2:file:data/db

2015-11-20 23:16:33,103 - [INFO] - from play in play-internal-execution-context-2 
Application started (Dev)

2015-11-20 23:19:03,473 - [INFO] - from play in play-internal-execution-context-4 
datasource [jdbc:h2:file:data/db;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-11-20 23:19:03,477 - [INFO] - from play in play-internal-execution-context-4 
database [default] connected at jdbc:h2:file:data/db

2015-11-20 23:19:03,648 - [INFO] - from play in play-internal-execution-context-4 
Application started (Dev)

2015-11-20 23:19:27,610 - [ERROR] - from play in play-internal-execution-context-4 
Cannot invoke the action, eventually got an error: scala.MatchError: null

2015-11-20 23:19:27,631 - [ERROR] - from application in play-internal-execution-context-4 


! @6o7g2hfnc - Internal server error, for (POST) [/climate/authenticate] ->

play.api.Application$$anon$1: Execution exception[[MatchError: null]]
	at play.api.Application$class.handleError(Application.scala:293) ~[play_2.10-2.2.3.jar:2.2.3]
	at play.api.DefaultApplication.handleError(Application.scala:399) [play_2.10-2.2.3.jar:2.2.3]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3$$anonfun$applyOrElse$3.apply(PlayDefaultUpstreamHandler.scala:264) [play_2.10-2.2.3.jar:2.2.3]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3$$anonfun$applyOrElse$3.apply(PlayDefaultUpstreamHandler.scala:264) [play_2.10-2.2.3.jar:2.2.3]
	at scala.Option.map(Option.scala:145) [scala-library-2.10.3.jar:na]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3.applyOrElse(PlayDefaultUpstreamHandler.scala:264) [play_2.10-2.2.3.jar:2.2.3]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3.applyOrElse(PlayDefaultUpstreamHandler.scala:260) [play_2.10-2.2.3.jar:2.2.3]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:344) [scala-library-2.10.3.jar:na]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:343) [scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library-2.10.3.jar:na]
	at play.api.libs.iteratee.Execution$$anon$1.execute(Execution.scala:43) [play-iteratees_2.10-2.2.3.jar:2.2.3]
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:40) [scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:248) [scala-library-2.10.3.jar:na]
	at scala.concurrent.Promise$class.complete(Promise.scala:55) [scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:153) [scala-library-2.10.3.jar:na]
	at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:235) [scala-library-2.10.3.jar:na]
	at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:235) [scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) [scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) [scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) [scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) [scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) [scala-library-2.10.3.jar:na]
Caused by: scala.MatchError: null
	at play.core.j.JavaAction$$anon$1$$anon$2.apply(JavaAction.scala:57) ~[play_2.10-2.2.3.jar:2.2.3]
	at play.core.j.JavaAction$$anon$1$$anon$2.apply(JavaAction.scala:56) ~[play_2.10-2.2.3.jar:2.2.3]
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82) ~[play_2.10-2.2.3.jar:2.2.3]
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82) ~[play_2.10-2.2.3.jar:2.2.3]
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library-2.10.3.jar:na]
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37) ~[play_2.10-2.2.3.jar:2.2.3]
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42) ~[akka-actor_2.10-2.2.0.jar:2.2.0]
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386) ~[akka-actor_2.10-2.2.0.jar:2.2.0]
	... 4 common frames omitted

2015-11-20 23:20:42,570 - [INFO] - from play in play-internal-execution-context-4 
datasource [jdbc:h2:file:data/db;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-11-20 23:20:42,574 - [INFO] - from play in play-internal-execution-context-4 
database [default] connected at jdbc:h2:file:data/db

2015-11-20 23:20:42,696 - [INFO] - from play in play-internal-execution-context-4 
Application started (Dev)

2015-11-20 23:20:42,735 - [ERROR] - from play in play-internal-execution-context-4 
Cannot invoke the action, eventually got an error: scala.MatchError: null

2015-11-20 23:20:42,738 - [ERROR] - from application in play-internal-execution-context-4 


! @6o7g2ln0e - Internal server error, for (POST) [/climate/authenticate] ->

play.api.Application$$anon$1: Execution exception[[MatchError: null]]
	at play.api.Application$class.handleError(Application.scala:293) ~[play_2.10-2.2.3.jar:2.2.3]
	at play.api.DefaultApplication.handleError(Application.scala:399) [play_2.10-2.2.3.jar:2.2.3]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3$$anonfun$applyOrElse$3.apply(PlayDefaultUpstreamHandler.scala:264) [play_2.10-2.2.3.jar:2.2.3]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3$$anonfun$applyOrElse$3.apply(PlayDefaultUpstreamHandler.scala:264) [play_2.10-2.2.3.jar:2.2.3]
	at scala.Option.map(Option.scala:145) [scala-library-2.10.3.jar:na]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3.applyOrElse(PlayDefaultUpstreamHandler.scala:264) [play_2.10-2.2.3.jar:2.2.3]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3.applyOrElse(PlayDefaultUpstreamHandler.scala:260) [play_2.10-2.2.3.jar:2.2.3]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:344) [scala-library-2.10.3.jar:na]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:343) [scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library-2.10.3.jar:na]
	at play.api.libs.iteratee.Execution$$anon$1.execute(Execution.scala:43) [play-iteratees_2.10-2.2.3.jar:2.2.3]
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:40) [scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:248) [scala-library-2.10.3.jar:na]
	at scala.concurrent.Promise$class.complete(Promise.scala:55) [scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:153) [scala-library-2.10.3.jar:na]
	at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:235) [scala-library-2.10.3.jar:na]
	at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:235) [scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) [scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) [scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) [scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) [scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) [scala-library-2.10.3.jar:na]
Caused by: scala.MatchError: null
	at play.core.j.JavaAction$$anon$1$$anon$2.apply(JavaAction.scala:57) ~[play_2.10-2.2.3.jar:2.2.3]
	at play.core.j.JavaAction$$anon$1$$anon$2.apply(JavaAction.scala:56) ~[play_2.10-2.2.3.jar:2.2.3]
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82) ~[play_2.10-2.2.3.jar:2.2.3]
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82) ~[play_2.10-2.2.3.jar:2.2.3]
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library-2.10.3.jar:na]
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37) ~[play_2.10-2.2.3.jar:2.2.3]
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42) ~[akka-actor_2.10-2.2.0.jar:2.2.0]
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386) ~[akka-actor_2.10-2.2.0.jar:2.2.0]
	... 4 common frames omitted

2015-11-20 23:23:08,827 - [INFO] - from play in play-internal-execution-context-4 
datasource [jdbc:h2:file:data/db;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-11-20 23:23:08,830 - [INFO] - from play in play-internal-execution-context-4 
database [default] connected at jdbc:h2:file:data/db

2015-11-20 23:23:08,999 - [INFO] - from play in play-internal-execution-context-4 
Application started (Dev)

2015-11-20 23:27:17,049 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:file:data/db;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-11-20 23:27:17,052 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:file:data/db

2015-11-20 23:27:17,199 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2015-11-20 23:56:24,884 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:file:data/db;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-11-20 23:56:24,887 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:file:data/db

2015-11-20 23:56:25,042 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2015-11-20 23:56:25,054 - [INFO] - from application in play-akka.actor.default-dispatcher-39 
http://localhost:9034/climateService/getAllClimateServices/json

2015-11-21 00:00:02,204 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:file:data/db;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-11-21 00:00:02,207 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:file:data/db

2015-11-21 00:00:02,356 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2015-11-21 00:00:02,367 - [INFO] - from application in play-akka.actor.default-dispatcher-41 
http://localhost:9034/climateService/getAllClimateServices/json

2015-11-21 00:00:03,630 - [INFO] - from application in play-akka.actor.default-dispatcher-49 
http://localhost:9034/climateService/getAllClimateServices/json

2015-11-21 00:00:04,505 - [INFO] - from application in play-akka.actor.default-dispatcher-52 
http://localhost:9034/climateService/getAllClimateServices/json

2015-11-21 00:14:32,263 - [INFO] - from play in play-internal-execution-context-9 
datasource [jdbc:h2:file:data/db;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-11-21 00:14:32,267 - [INFO] - from play in play-internal-execution-context-9 
database [default] connected at jdbc:h2:file:data/db

2015-11-21 00:14:32,437 - [INFO] - from play in play-internal-execution-context-9 
Application started (Dev)

2015-11-21 00:21:50,809 - [INFO] - from application in play-akka.actor.default-dispatcher-61 
http://localhost:9034/climateService/getAllClimateServices/json

2015-11-21 00:40:44,788 - [ERROR] - from application in New I/O worker #8 


! @6o7gd92gg - Internal server error, for (GET) [/climate] ->

play.PlayExceptions$CompilationException: Compilation error[not enough arguments for method apply: (form: play.data.Form[models.User])play.api.templates.HtmlFormat.Appendable in object header.
Unspecified value parameter form.]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10-2.2.3.jar:2.2.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10-2.2.3.jar:2.2.3]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.10.3.jar:na]

2015-11-21 00:49:27,323 - [ERROR] - from application in New I/O worker #1 


! @6o7geclg4 - Internal server error, for (GET) [/climate] ->

play.PlayExceptions$CompilationException: Compilation error[identifier expected but 'class' found.]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10-2.2.3.jar:2.2.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10-2.2.3.jar:2.2.3]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.10.3.jar:na]

2015-11-21 00:49:29,541 - [ERROR] - from application in New I/O worker #1 


! @6o7gecond - Internal server error, for (GET) [/climate] ->

play.PlayExceptions$CompilationException: Compilation error[identifier expected but 'class' found.]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10-2.2.3.jar:2.2.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10-2.2.3.jar:2.2.3]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.10.3.jar:na]

2015-11-21 00:49:45,426 - [ERROR] - from application in New I/O worker #1 


! @6o7gedmac - Internal server error, for (GET) [/climate] ->

play.PlayExceptions$CompilationException: Compilation error[not found: value form]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10-2.2.3.jar:2.2.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10-2.2.3.jar:2.2.3]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.10.3.jar:na]

2015-11-21 00:49:55,797 - [ERROR] - from application in New I/O worker #1 


! @6o7geebja - Internal server error, for (GET) [/climate] ->

play.PlayExceptions$CompilationException: Compilation error[object models.User is not a value]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10-2.2.3.jar:2.2.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10-2.2.3.jar:2.2.3]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.10.3.jar:na]

2015-11-21 00:50:58,008 - [INFO] - from play in play-internal-execution-context-9 
datasource [jdbc:h2:file:data/db;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-11-21 00:50:58,012 - [INFO] - from play in play-internal-execution-context-9 
database [default] connected at jdbc:h2:file:data/db

2015-11-21 00:50:58,119 - [INFO] - from play in play-internal-execution-context-9 
Application started (Dev)

2015-11-21 00:50:58,137 - [INFO] - from application in play-akka.actor.default-dispatcher-66 
http://localhost:9034/climateService/getAllClimateServices/json

2015-11-21 00:50:58,464 - [INFO] - from play in play-internal-execution-context-9 
datasource [jdbc:h2:file:data/db;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-11-21 00:50:58,467 - [INFO] - from play in play-internal-execution-context-9 
database [default] connected at jdbc:h2:file:data/db

2015-11-21 00:50:58,528 - [INFO] - from play in play-internal-execution-context-9 
Application started (Dev)

2015-11-21 00:51:30,244 - [INFO] - from play in play-internal-execution-context-9 
datasource [jdbc:h2:file:data/db;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-11-21 00:51:30,264 - [INFO] - from play in play-internal-execution-context-9 
database [default] connected at jdbc:h2:file:data/db

2015-11-21 00:51:30,418 - [INFO] - from play in play-internal-execution-context-9 
Application started (Dev)

2015-11-21 00:51:30,429 - [INFO] - from application in play-akka.actor.default-dispatcher-73 
http://localhost:9034/climateService/getAllClimateServices/json

2015-11-21 00:51:38,052 - [INFO] - from application in play-akka.actor.default-dispatcher-75 
http://localhost:9034/climateService/getAllClimateServices/json

2015-11-21 00:53:13,517 - [INFO] - from play in play-internal-execution-context-9 
datasource [jdbc:h2:file:data/db;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-11-21 00:53:13,522 - [INFO] - from play in play-internal-execution-context-9 
database [default] connected at jdbc:h2:file:data/db

2015-11-21 00:53:13,684 - [INFO] - from play in play-internal-execution-context-9 
Application started (Dev)

2015-11-21 00:53:13,694 - [INFO] - from application in play-akka.actor.default-dispatcher-76 
http://localhost:9034/climateService/getAllClimateServices/json

2015-11-21 00:53:15,929 - [INFO] - from application in play-akka.actor.default-dispatcher-84 
http://localhost:9034/climateService/getAllClimateServices/json

2015-11-21 00:53:23,060 - [INFO] - from application in play-akka.actor.default-dispatcher-85 
http://localhost:9034/climateService/getAllClimateServices/json

2015-11-21 01:14:11,709 - [ERROR] - from application in New I/O worker #6 


! @6o7ghj7bn - Internal server error, for (GET) [/climate] ->

play.PlayExceptions$CompilationException: Compilation error[not found: value form]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10-2.2.3.jar:2.2.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10-2.2.3.jar:2.2.3]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.10.3.jar:na]

2015-11-21 01:14:29,052 - [INFO] - from play in play-internal-execution-context-13 
datasource [jdbc:h2:file:data/db;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-11-21 01:14:29,055 - [INFO] - from play in play-internal-execution-context-13 
database [default] connected at jdbc:h2:file:data/db

2015-11-21 01:14:29,274 - [INFO] - from play in play-internal-execution-context-13 
Application started (Dev)

2015-11-21 01:14:29,286 - [INFO] - from application in play-akka.actor.default-dispatcher-82 
http://localhost:9034/climateService/getAllClimateServices/json

2015-11-21 01:14:40,601 - [INFO] - from application in play-akka.actor.default-dispatcher-95 
http://localhost:9034/climateService/getAllClimateServices/json

2015-11-21 01:22:00,669 - [INFO] - from application in play-akka.actor.default-dispatcher-105 
http://localhost:9034/climateService/getAllClimateServices/json

2015-11-21 01:22:17,833 - [INFO] - from application in play-akka.actor.default-dispatcher-102 
http://localhost:9034/climateService/getAllClimateServices/json

2015-11-21 01:22:28,523 - [INFO] - from application in play-akka.actor.default-dispatcher-102 
http://localhost:9034/climateService/getAllMostRecentClimateServicesByLatestAccessTime/json

2015-11-21 01:22:39,153 - [INFO] - from application in play-akka.actor.default-dispatcher-111 
http://localhost:9034/climateService/getAllClimateServices/json

2015-11-21 01:23:41,487 - [INFO] - from application in play-akka.actor.default-dispatcher-111 
http://localhost:9034/climateService/getAllMostRecentClimateServicesByLatestAccessTime/json

2015-11-21 01:28:34,839 - [INFO] - from application in play-akka.actor.default-dispatcher-111 
http://localhost:9034/climateService/getAllClimateServices/json

2015-11-21 01:39:23,548 - [ERROR] - from application in New I/O worker #5 


! @6o7gl17ng - Internal server error, for (GET) [/climate] ->

play.PlayExceptions$CompilationException: Compilation error[error: missing return statement]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10-2.2.3.jar:2.2.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10-2.2.3.jar:2.2.3]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.10.3.jar:na]

2015-11-21 01:40:20,101 - [INFO] - from play in play-internal-execution-context-15 
datasource [jdbc:h2:file:data/db;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-11-21 01:40:20,104 - [INFO] - from play in play-internal-execution-context-15 
database [default] connected at jdbc:h2:file:data/db

2015-11-21 01:40:20,256 - [INFO] - from play in play-internal-execution-context-15 
Application started (Dev)

2015-11-21 01:40:20,268 - [INFO] - from application in play-akka.actor.default-dispatcher-121 
http://localhost:9034/climateService/getAllClimateServices/json

2015-11-21 01:40:38,012 - [INFO] - from play in play-internal-execution-context-15 
datasource [jdbc:h2:file:data/db;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-11-21 01:40:38,016 - [INFO] - from play in play-internal-execution-context-15 
database [default] connected at jdbc:h2:file:data/db

2015-11-21 01:40:38,144 - [INFO] - from play in play-internal-execution-context-15 
Application started (Dev)

2015-11-21 01:40:38,155 - [INFO] - from application in play-akka.actor.default-dispatcher-132 
http://localhost:9034/climateService/getAllClimateServices/json

2015-11-21 01:42:48,720 - [INFO] - from application in play-akka.actor.default-dispatcher-127 
http://localhost:9034/climateService/getAllClimateServices/json

2015-11-21 01:43:10,656 - [INFO] - from application in play-akka.actor.default-dispatcher-141 
http://localhost:9034/climateService/getAllClimateServices/json

2015-11-21 01:45:23,906 - [ERROR] - from application in New I/O worker #1 


! @6o7gllkpf - Internal server error, for (GET) [/climate] ->

play.PlayExceptions$CompilationException: Compilation error[error: ';' expected]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10-2.2.3.jar:2.2.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10-2.2.3.jar:2.2.3]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.10.3.jar:na]

2015-11-21 01:45:44,675 - [INFO] - from play in play-internal-execution-context-17 
datasource [jdbc:h2:file:data/db;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-11-21 01:45:44,681 - [INFO] - from play in play-internal-execution-context-17 
database [default] connected at jdbc:h2:file:data/db

2015-11-21 01:45:44,835 - [INFO] - from play in play-internal-execution-context-17 
Application started (Dev)

2015-11-21 01:45:44,846 - [INFO] - from application in play-akka.actor.default-dispatcher-139 
http://localhost:9034/climateService/getAllClimateServices/json

2015-11-21 01:45:58,655 - [INFO] - from application in play-akka.actor.default-dispatcher-151 
http://localhost:9034/climateService/getAllClimateServices/json

2015-11-21 01:48:13,853 - [INFO] - from application in play-akka.actor.default-dispatcher-153 
http://localhost:9034/climateService/getAllClimateServices/json

2015-11-21 01:48:18,693 - [INFO] - from play in play-internal-execution-context-17 
datasource [jdbc:h2:file:data/db;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-11-21 01:48:18,695 - [INFO] - from play in play-internal-execution-context-17 
database [default] connected at jdbc:h2:file:data/db

2015-11-21 01:48:18,844 - [INFO] - from play in play-internal-execution-context-17 
Application started (Dev)

2015-11-21 01:48:43,760 - [INFO] - from application in play-akka.actor.default-dispatcher-159 
http://localhost:9034/climateService/getAllClimateServices/json

2015-11-21 01:49:00,832 - [INFO] - from application in play-akka.actor.default-dispatcher-166 
http://localhost:9034/climateService/getAllClimateServices/json

2015-11-21 02:11:36,405 - [INFO] - from play in play-internal-execution-context-17 
datasource [jdbc:h2:file:data/db;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-11-21 02:11:36,407 - [INFO] - from play in play-internal-execution-context-17 
database [default] connected at jdbc:h2:file:data/db

2015-11-21 02:11:36,526 - [INFO] - from play in play-internal-execution-context-17 
Application started (Dev)

2015-11-21 02:11:36,534 - [INFO] - from application in play-akka.actor.default-dispatcher-166 
http://localhost:9034/climateService/getAllClimateServices/json

2015-11-21 02:12:38,061 - [INFO] - from play in play-internal-execution-context-20 
datasource [jdbc:h2:file:data/db;DB_CLOSE_DELAY=-1] bound to JNDI as DefaultDS

2015-11-21 02:12:38,063 - [INFO] - from play in play-internal-execution-context-20 
database [default] connected at jdbc:h2:file:data/db

2015-11-21 02:12:38,318 - [INFO] - from play in play-internal-execution-context-20 
Application started (Dev)

2015-11-21 02:13:59,826 - [INFO] - from application in play-akka.actor.default-dispatcher-191 
http://localhost:9034/climateService/getAllClimateServices/json

